test_unstructured_ingest/dest/pinecone.sh

```diff
@@ -93,7 +93,7 @@ PYTHONPATH=. ./unstructured/ingest/main.py \
   --work-dir "$WORK_DIR" \
   --chunk-elements \
   --chunk-combine-text-under-n-chars 200 --chunk-new-after-n-chars 2500 --chunk-max-characters 38000 --chunk-multipage-sections \
-  --embedding-provider "langchain-huggingface" \
+  --embedding-provider "langchain-vertexai" \
```

Comment:
might be better to leave this one as huggingface and do the embedding test like 
`test_unstructured_ingest/src/local-embed-bedrock.sh` 
Thoughts?

Reply:
Was going to revert after a job tests this, that was my way of testing it basically :) 

Now added the tests with [add vertexai integration test](https://github.com/Unstructured-IO/unstructured/pull/2693/commits/43bcdfc705a9ee442994bfcbcfee4d1f551c36ef) and [add octoai test](https://github.com/Unstructured-IO/unstructured/pull/2693/commits/66f1a2c8bb7f3889c536f0bdb7320bee65855e3e)

Reply:
Awesome! Thanks for doing the octoai :) Was on my list of todos...

Reply:
Sure thing 💪 🚀